docs/source/qvm.rst

```diff
@@ -179,11 +179,11 @@ The Quantum Processing Unit (QPU)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To access a QPU endpoint, you will have to `sign up <https://www.rigetti.com/>`_ for Quantum Cloud Services (QCS).
-Documentation for getting started can be found `here <https://www.rigetti.com/qcs/docs/intro-to-qcs>`_. The
-documentation will show you how to reserve time on a QPU. When your reservation begins, you will be authorized to
-access the QPU, and you can submit requests using pyQuil.
+Documentation for getting started can be found `here <https://www.rigetti.com>`_. The documentation will show you how to
```

Comment:
I would change this link to https://docs.rigetti.com

Reply:
Good catch!